### PR TITLE
Closes #1400 - edits to renal intercalated cell types

### LINKED ISSUE #1400: renal beta-intercalated cell should be part of collecting duct of renal tubule
Poss supporting refs (from ASCT+B table version 1.1)
PMID: 31249312. PMCID: PMC6597610	DOI: 10.1038/s41467-019-10861-2		PMID: 31689386	DOI: 10.1016/j.devcel.2019.10.005

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Closes #1400 - edits to renal intercalated cell types

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
Hi, @dosumis, @paolaroncaglia, this PR is ready for review.

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -21531,6 +21535,7 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CL:CVS"^^xsd:string) obo:IAO_
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0005010 "FMA:86560"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0005010 "intercalated cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0005010 "cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25632105"^^xsd:string) rdfs:comment obo:CL_0005010 "Intercalated cells are also called “mitochondria-rich cells,” reflecting the high levels of round mitochondria at their apical pole, or in their cytoplasm, a distribution that was quite different from that of other kidney tubule cells, which accumulated mitochondria around the basolateral membrane. These cells also have “numerous irregular apical microvilli” compared with the surrounding segment-specific cells. Intercalated cells lack a central cilium, at least in the cortex, which also differentiates them from the adjacent principal cells.")
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:21538]: “mitochondria-rich cells,” => “mitochondria-rich cells”,
a distribution that was quite different => ...that is...
These cells also have “numerous irregular apical microvilli” => I'd remove the quotation marks

**Review by paolaroncaglia — COMMENTED** [review]
@bvarner-ebi I added some minor comments, otherwise looks good. Thanks!

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:21538]: Thank you for the review, @paolaroncaglia!
Edits made.
Of note, I believe the standard for this ontology is to use the American spelling variants. Does that standard extend to punctuation as well? For your first suggestion, I believe the American standard places the comma inside the quotation marks and the British standard would place it outside. I selected the latter as per your suggestion.

Comment (paolaroncaglia) [reply src/ontology/cl-edit.owl:21538]: @bvarner-ebi yes, standard for GO and some other ontologies for sure is to use US spelling. I've not been advised on punctuation specifically, but TBH, I've never come across punctuation inside quotation marks in the 4 ontologies I've worked on. English is not my first language, but my first language follows the same standard as the British one - commas inside quotation marks would be considered a (minor) error. :-)

**Review by paolaroncaglia — APPROVED** [review]

**Pull request merged**